src/app/home/home.component.html

```diff
@@ -33,14 +33,7 @@
       <div class="row">
         <img src= "assets/images/homescreenimage.png" id="home-img">
       </div>
-
-      <div class="row">
-         <div class="col-sm-8 col-md-6 col-md-offset-3 col-sm-offset-2">
-            <button (click)="patientOnClick()" type="button" class = "btn btn-success">Patient Check In</button>
-        </div>
```

Comment:
Why was this removed?  PR #69 will change where it routes to.  Maybe wait until that one goes through, and then rebase.

Reply:
There wasn't a real reason I removed it. I figured since the component was ripped out the button shouldn't be there either. I can add it back in.